packages/cli/tests/create.test.js

```diff
@@ -29,6 +29,15 @@ describe('preact create', () => {
 		expect(template.includes('twitter:card')).toEqual(true);
 	});
 
+	it(`should have 'apple-touch-icon' meta tag`, async () => {
```

Comment:
This test should probably. test more things in head rather than just an apple meta tag